lib/fountain/api/funnels.rb

```diff
@@ -18,6 +18,24 @@ def self.list(list_options = {})
         Fountain::Funnels.new response
       end
 
+      #
+      # Get Funnel Info
+      # @param [String] funnel_id ID of the Fountain funnel
+      # @return [Fountain::Funnel]
+      def self.get(funnel_id)
+        response = request_json("/v2/funnels/#{funnel_id}")
+        Fountain::Funnel.new response
+      end
+
+      #
+      # Get Stage Info
+      # @param [String] stage_id ID of the Fountain stage
+      # @return [Fountain::Stage]
+      def self.get_stage(stage_id)
+        response = request_json("/v2/stages/#{stage_id}")
+        Fountain::Stage.new response
+      end
```

Comment:
looks like this would better live in `lib/fountain/api/stages.rb` (new file) with method signature `def self.get(stage_id)`, although the implementation would stay the same. Are you ok to make the change?